Error loading formatter when running Sabre command line tool

## Description

The Sabre CLI fails with an error message when installed via npm:

```
$ sabre simple_suicide.sol 
✔ Compiled with solc v0.5.7 successfully
✖ Analysis failed
Error: There was a problem loading formatter: /Users/bernhardmueller/Library/Mobile Documents/com~apple~CloudDocs/MythX-Testing/lib/formatters/text
Error: Cannot find module '/Users/bernhardmueller/Library/Mobile Documents/com~apple~CloudDocs/MythX-Testing/lib/formatters/text'
```

##

The command line tool should work from any directory.

--- a/lib/report.js
+++ b/lib/report.js
@@ -1,3 +1,4 @@
+const path = require('path');
 const eslintHelpers = require('./eslint');
 const eslintCliEngine = require("eslint").CLIEngine;
 const SourceMappingDecoder = require('remix-lib/src/sourceMappingDecoder');
@@ -16,7 +17,7 @@ const getFormatter = (name) => {
     const custom = ['text'];
 
     if (custom.includes(name)) {
-        name = './lib/formatters/' + name;
+        name = path.join(__dirname, 'formatters/', name + '.js');
     }
 
     return eslintCliEngine.getFormatter(name);

